src/test/java/marquez/core/models/ModelEqualsAndHashCode.java

```diff
@@ -0,0 +1,61 @@
+package marquez.core.models;
```

Comment:
If it's a test class, please use the naming convention `XYZTest.java`.

Reply:
great catch, thanks

Reply:
removed this file